lib/browser/api/menu.js

```diff
@@ -118,7 +119,11 @@ Menu.prototype._init = function () {
     executeCommand: (event, commandId) => {
       const command = this.commandsMap[commandId]
       if (command == null) return
-      command.click(event, BrowserWindow.getFocusedWindow(), webContents.getFocusedWebContents())
+      // If there was a window passed to `popup()` method, use it.
+      // Otherwise use a focused one.
+      const menuWindow = this.window || BrowserWindow.getFocusedWindow()
+      const menuWebContents = this.window ? this.window.webContents : webContents.getFocusedWebContents()
```

Comment:
I think this might break the context menu in the console tab of the dev tools on Linux and Windows.

I built this branch and tried cut/copy/paste in the dev tools Console tab on Windows 10 and the menu items did not seem to work.

This is because the focused web contents returns the dev tools web contents but `this.window.webContents` will be the web contents of the page, not the dev tools as it should be when the dev tools is open and focused.

The dev tools context menu passed in the window here: https://github.com/electron/electron/blob/1f75df0253c7cee435f2783029ed26ea20ea7ad0/lib/renderer/inspector.js#L66